src/Arduino/EssaimRobots/Controllers.h

```diff
@@ -87,28 +86,32 @@ extern  Zumo32U4Motors  motors;
                                         *   '<S38>/Proportional Gain'
                                         *   '<S82>/Proportional Gain'
                                         */
-#define Vm  5.0                         // Variable: Vm 
+#define Vm  40                       // Variable: Vm cm/s
 
-typedef float real32_T;
+typedef float real_T;
+
+/* Block signals for system '<Root>/Robot_controller' */
+typedef struct {
+  real_T addantiwindup;                /* '<S1>/Saturation1' */
+} B_Robot_controller_decouplant_T;
 
 /* Block states (default storage) for system '<Root>/Robot_controller' */
 typedef struct {
-  real32_T DiscreteTimeIntegrator_DSTATE;/* '<S1>/Discrete-Time Integrator' */
-  real32_T Integrator_DSTATE;            /* '<S33>/Integrator' */
-  real32_T Filter_DSTATE;                /* '<S28>/Filter' */
-  real32_T Integrator_DSTATE_g;          /* '<S77>/Integrator' */
-  real32_T Filter_DSTATE_c;              /* '<S72>/Filter' */
+  real_T DiscreteTimeIntegrator_DSTATE;/* '<S1>/Discrete-Time Integrator' */
+  real_T Integrator_DSTATE;            /* '<S33>/Integrator' */
+  real_T Filter_DSTATE;                /* '<S28>/Filter' */
+  real_T Integrator_DSTATE_g;          /* '<S77>/Integrator' */
+  real_T Filter_DSTATE_c;              /* '<S72>/Filter' */
 } DW_uncouping_controller_T;
```

Comment:
I suggest renaming these structs to "uncouping_controller_signals" and "uncoupling_controller_states" respectively.